.rubocop.yml

```diff
@@ -14,4 +15,10 @@ Layout/DotPosition:
   EnforcedStyle: trailing
 
 Style/BracesAroundHashParameters:
-  Enabled: false
\ No newline at end of file
+  Enabled: false
```

Comment:
I vote this one is enabled.  It helps prevent accidental and weird handling of parameters passed into methods.

Reply:
I agree, but this was pre-existing, I just added a newline (hence the change). So I'm not going to change strategy in this PR.